app/build.gradle

```diff
@@ -293,6 +299,11 @@ android {
             // use the settings from defaultConfig
         }
 
+        managed {
```

Comment:
What does this build flavor represent? I'm not following why this implementation would need something like this